### PR TITLE
DX-2081: Added contact us and partners

### PR DESCRIPTION
**Pull request by arebra**
Added card for sales contact and partners. Also a landing page for the different countries' partners.

### CODE REVIEW

File: partners.md
Comment (OyvindMathisen) [inline partners.md:1]: **Question, Minor**

Is it correct that the `partner` button should only appear after clicking the field on the main landing page?

Before
Before

After
After

Comment (OyvindMathisen) [inline partners.md:1]: **Question, Minor**

Assuming you got no control over the sidebar here?
Looks a bit weird to have the main title and subtitle, where both leads to just the same page.
Maybe it's just me though.

image

File: partners.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+              to='https://www.swedbankpay.no/partners'
+          %}
+      </div>
+            <div class="{{ card_col_class }}">
+          {% include card.html title='Swedish'
+              text='Our Swedish partners'
+              icon_content='email'
+              to='https://www.swedbankpay.se/partners'
+          %}
+      </div>
```
Comment (OyvindMathisen) [inline partners.md:36]: **Nitpick, Minor**

Variable indentation. Consider lining it up?

```suggestion
      <div class="{{ card_col_class }}">
          {% include card.html title='Denmark'
              text='Our Danish partners'
              icon_content='email'
              icon_outlined=true
              to='https://www.swedbankpay.dk/partners'
          %}
      </div>
      <div class="{{ card_col_class }}">
          {% include card.html title='Norway'
              text='Our Norwegian partners'
              icon_content='email'
              to='https://www.swedbankpay.no/partners'
          %}
      </div>
      <div class="{{ card_col_class }}">
          {% include card.html title='Swedish'
              text='Our Swedish partners'
              icon_content='email'
              to='https://www.swedbankpay.se/partners'
          %}
      </div>
```

Comment (OyvindMathisen) [inline partners.md:36]: **Nitpick, Minor**

Denmark has a white background on the logo, while Norway and Sweden has a dark background? Intended?

image

Comment (OyvindMathisen) [reply partners.md:36]: Denmark seems to have `icon_outlined=true` while Norway and Sweden does not.

Comment (OyvindMathisen) [inline partners.md:36]: **Question, Minor**

Denmark and Norway refers to the country, while Sweden seems to related to the language?
Shouldn't they all be the same, either country or language?

I.E.
Denmark, Norway, Sweden
or
Danish, Norwegian, Swedish

**Review by OyvindMathisen — APPROVED** [review]
A couple of questions, but nothing major or breaking.
Ran it locally and it looked fine to me.

Look over the comments if you can/have time, but nothing to important if you're in a hurry.

File: partners.md
Comment (arebra) [reply partners.md:1]: Yes, it is hidden from sidebar unless you are on the actual partner page. It does not seem important enough to be a visible point on the menu, but I needed a landing page for the card as there are three different partner pages depending on the country. Maybe I can add it under the resources instead.

File: partners.md (hunk 1 above)
Comment (arebra) [reply partners.md:36]: Good catch! I'll fix it :)

Comment (arebra) [reply partners.md:36]: Thanks! A lack of concentration on my part. You are absolutely right.

**Pull request merged**